src/main/java/org/sonar/report/pdf/builder/RuleBuilder.java

```diff
@@ -93,7 +93,10 @@ public void loadViolatedResources(final Rule rule, final String ruleKey, final S
       // ruleKey = URLEncoder.encode(ruleKey, "UTF8");
       LOG.debug("Accessing Sonar: getting violated resurces by one given rule (" + ruleKey + ")");
 
-      SonarClient client = SonarClient.create(credentials.getUrl());
+      SonarClient client = SonarClient.builder().url(credentials.getUrl())
+                                                .login(credentials.getUsername())
```

Comment:
It would be the same as not passing the parameter, since on SonarClient both username and password are null by default. Should I add a conditional check even so, just in case that changes in the future?